### ISSUE TITLE
Vacation settings box cut off at bottom of screen

### ISSUE THREAD

**Issue by noamross** [leggett/simplify#820]
My dialog box for vacation settings, which appears when vacation mode is on, is cut off.  This yellow box always appears at the bottom of the window, placed so that half the box and text is below the window edge.  

Here is a screen shot of my gmail at the bottom edge of the window:

<img width="444" alt="Screenshot 2024-07-08 at 5 19 41 PM" src="https://github.com/leggett/simplify/assets/571752/2872f798-59e2-4a68-af4a-02dba0ec910c">

Here is my system report from the Simplify menu:

```
Simplify v.3.1.14 configuration: aAX caA caI caL cfC cfF cfO cfR cfU composeActions composeFab defaultTheme fontList fontMsg fontNav geminiMin genSumMin hideAds hideEmptySections hideFileIcons hideImportance hideInboxHideAllLink hideLabelChips hideListCount hideMsgCount hideMsgReactions hideMsgReplies hideNavUnreads hideSignatures hideStars hideTabIcons highDensity inInbox inList invertCompose isFirefox izBgOff lightTheme listWidthLg matYou msgOpen msgWidthSm navOpen newNav newUI reminders rightImportance rightLabels rightStar sendLater simplify simplifyPrefsOpen vPane- 

System: Mozilla/5.0 (Macintosh Intel Mac OS X 10.15 rv:127.0) Gecko/20100101 Firefox/127.0 - gmail.pinto-server_20240624.05_p0 - Window: 2042 x 1438 (113pct zoom) - Language: en - Errors: / 9513:92: Couldn't find profile button to build App Menu. Was loading = true / 1936:71: get(.a4W) called with undefined parent, null / 2221:54: clickOnWhenReady(.vh div[role="button"]) timed out
```

I have none of the extensions listed in any of the groups in https://github.com/leggett/simplify/issues/554 . I don't believe any of my other extensions modify Gmail CSS/appearance.

**Comment by leggett** [comment]
Thanks for the report -- it looks like Gmail changed something and this modification has broken. I'll work to get it fixed in the next update.

**Comment by leggett** [comment]
Fix for this is finally going out in v3.1.15. Thanks for the report.
